### ISSUE TITLE
[dataio] dataio-pyshovel goto frame offset (Trac #1508)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1500]
goto frame uses frame index starting at 0, but the display uses frame index starting at 1. this leads to an offset when trying to go to a specific frame number.

Migrated from https://code.icecube.wisc.edu/ticket/1508
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:22", 
    "description": "goto frame uses frame index starting at 0, but the display uses frame index starting at 1. this leads to an offset when trying to go to a specific frame number.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067082284240", 
    "component": "combo core", 
    "summary": "[dataio] dataio-pyshovel goto frame offset", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-01-13T16:29:48", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160205T21:56:05: david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160205T23:08:26: 
* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 2400:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2400"
> fix goto frame offset. closes #1500
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:51:15: nwhitehorn commented:

> In changeset 169184:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169184"
> fix goto frame offset. closes #1500
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:22: 
* olivas commented:

> Milestone March 2016 deleted


* olivas changed milestone from "March 2016" to ""
